Populate the timeline description from the README.md (#49)
googleinterns/step189-2020: README.md

## Timeline

> ### Demo
> ![Timeline demo](/images/timeline.gif?raw=true "GIF of timeline")

Reviewer: ```suggestion
### Demo

![Timeline demo](/images/timeline.gif?raw=true "GIF of timeline")
```

### Description

> The timeline tackles the first of our big questions: how do we compare pushes against each other? To solve this problem, we place all the pushes onto a timeline using the push’s start and end values to define its corresponding length. Placing pushes side by side allows users to easily spot patterns in their push history and efficiently zero in on areas where those patterns are broken.

Reviewer 2: Maybe change the 'push’s start and end values' to push’s start and end timestamps'?

> Intervals on the timeline are sorted by start time and optimally grouped into rows such that the maximum number of non-overlapping intervals are placed within one row. This ensures that the timeline is as vertically compact as possible.The color of the interval corresponds to the final stage of each push. Hovering over the interval produces a tooltip display with the push’s push ID, ending stage, start time, end time, and push duration (given in a human readable format).

Reviewer: ```suggestion
The intervals on the timeline are sorted by start time and optimally grouped into rows such that the maximum number of non-overlapping intervals are placed within one row. This ensures that the timeline is as vertically compact as possible. The color of an interval corresponds to the final stage of each push. Hovering over an interval produces a tooltip display with the push’s push ID, ending stage, start time, end time, and push duration (given in a human readable format).
```

> With regards to UI design, the goal was to keep the timeline as minimal and clean as possible while also conveying the maximum amount of data. The timeline has standard horizontal panning and zooming, which are both limited to the given domain. To clearly indicate hovering, the selected interval will drop in opacity while also gaining a noticeable gray border. To combat the problem of having miniscule pushes get lost in the sea of longer pushes, we set a thin border around each interval such that, even when zoomed all the way out, smaller pushes will still be visible. We also maintain the x-position by keeping a vertical line that tracks the movement of the mouse and has a corresponding label indicating the current x-axis value.

Reviewer: ```suggestion
With regards to UI design, the goal was to keep the timeline as minimal and clean as possible while also conveying the maximum amount of data. The timeline has standard horizontal panning and zooming, which are both limited to the given domain. To clearly indicate hovering, the selected interval will drop in opacity while also gaining a noticeable gray border. To combat the problem of having miniscule pushes get lost in the sea of longer pushes, we set a thin border around each interval such that, even when zoomed all the way out, smaller pushes will still be visible. We also highlight the current x-position by showing a vertical line that tracks the movement of the mouse and has a corresponding label indicating the current x-axis value.
```

Reviewer: I'm not sure why the diff doesn't work properly in this case. The changes I made only touch the last sentence.

The intervals on the timeline are sorted by start time and optimally grouped into rows such that the maximum number of non-overlapping intervals are placed within one row. This ensures that the timeline is as vertically compact as possible. The color of an interval corresponds to the final stage of each push. Hovering over an interval produces a tooltip display with the push’s push ID, ending stage, start time, end time, and push duration (given in a human readable format).

With regards to UI design, the goal was to keep the timeline as minimal and
clean as possible while also conveying the maximum amount of data. The timeline
has standard horizontal panning and zooming, which are both limited to the given
domain. To clearly indicate hovering, the selected interval will drop in opacity
while also gaining a noticeable gray border. To combat the problem of having
miniscule pushes get lost in the sea of longer pushes, we set a thin border
around each interval such that, even when zoomed all the way out, smaller pushes
will still be visible. We also highlight the current x-position by showing a
vertical line that tracks the movement of the mouse and has a corresponding

> label indicating the current x-axis value.

Reviewer: The rest of the paragraphs are a single lines. I think that was not a great decision that I started by copy/pasting the text from the Google Docs. I think we should be consistent and maybe wrap everything after all the text is in.
